wemake_python_styleguide/visitors/ast/exceptions.py

```diff
@@ -47,16 +49,45 @@ def _find_returing_nodes(
     return try_has, except_has, else_has, finally_has
 
 
+def _traverse_exception(
+    cls,
+    builtin_exceptions=None,
+) -> Dict[str, Tuple[str]]:
+    """
+    Orinigal code: see below.
+
+    https://github.com/thg-consulting/inspectortiger/blob/
+    6b9736b22528493cd41d0bf92f9fdcdd6c7cd129/inspectortiger/plugins/misc.py#L66
+    """
+    builtin_exceptions = builtin_exceptions or {}
+
+    if cls.__name__ not in builtin_exceptions:
+        builtin_exceptions[cls.__name__] = ()
+
+    for exc in cls.__subclasses__():
+        builtin_exceptions[exc.__name__] = tuple(
+            base.__name__
+            for base in getmro(exc)
+            if (
+                issubclass(base, BaseException) and
+                base.__name__ != exc.__name__
+            )
+        )
+        _traverse_exception(exc, builtin_exceptions)
+
+    return builtin_exceptions.copy()
```

Comment:
Why do we need to `.copy` it?

Reply:
Apparently, we don't need it. I'm gonna fix it now.